challenge-Programs/performance-challenge.md

```diff
@@ -91,3 +93,22 @@ How to exchange: Fill out the Redeem Gifts form after the end of this season (Th
 
 **Good luck and look forward to your brilliant idea!!**
 
+## FAQ (Updating)
+
+### Q1: How to change the registration information? How about the change of the Contribution Credit after the registration information changes?
+If you want to change the signup information, you need to close the wrong signup Issue and open a new one for signing up. Editing on the original Issue is unavailable for the moment. There are three kinds of changes related to the Contribution Credit after the change of registration information:
+
+- The total Contribution Credit owned by a team will not be affected by the change of team members;
+- If you join a team, your individual Contribution Credit will remain at the state before joining the team, and all the Contribution Credit you gained after joining the team will be owned by the team;
+- If you quit a team, your original Contribution Credit is "0" because you can't inherit the team Contribution Credit.
```

Comment:
What if the has credits before joining the team and now he quits to be individual again? He should still have the original credit according to point 1. @zz-jason

Reply:
Yes, "your original Contribution Credit is '0' " will be deleted. Thank you, Calvin!